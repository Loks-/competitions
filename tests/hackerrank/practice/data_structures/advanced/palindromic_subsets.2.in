15 15
bbbaccaccbacbaa
1 4 13 31
2 2 3
2 0 10
1 0 14 71
1 1 7 26
1 6 14 43
1 0 1 5
2 2 9
1 8 10 34
1 7 13 40
2 1 10
2 8 12
1 0 3 54
1 7 14 60
2 1 2
