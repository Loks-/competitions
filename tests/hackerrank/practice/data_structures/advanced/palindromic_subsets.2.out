2
383
27
19
5
2
